pom.xml

```diff
@@ -203,9 +202,9 @@
 				</configuration>
 			</plugin>
 			<plugin>
-				<groupId>org.mortbay.jetty</groupId>
-				<artifactId>jetty-maven-plugin</artifactId>
-				<version>8.1.14.v20131031</version>
+				<groupId>org.eclipse.jetty</groupId>
```

Comment:
Don't change this version (it should use the one I added a few minutes ago to the main FINRAOS repo)